app/modules/conversations/conversation/conversation_service.py

```diff
@@ -664,6 +665,17 @@ async def get_conversation_info(
                 .filter_by(conversation_id=conversation_id, status=MessageStatus.ACTIVE)
                 .count()
             )
+
+            agent_id = conversation.agent_ids[0] if conversation.agent_ids else None
+
+            agent_ids = conversation.agent_ids
+            if agent_id:
+                custom_agent = (
+                    self.sql_db.query(CustomAgent).filter_by(id=agent_id).first()
```

Comment:
can we not check if the agent id is a system id before querying? That would avoid so many db calls